gnoland/website/main.go

```diff
@@ -29,14 +29,16 @@ import (
 
 var flags struct {
 	bindAddr   string
-	remoteAddr string	
+	remoteAddr string
+	captcha	   bool	
 }
 
 var startedAt time.Time
 
 func init() {
 	flag.StringVar(&flags.remoteAddr, "remote", "127.0.0.1:26657", "remote gnoland node address")
 	flag.StringVar(&flags.bindAddr, "bind", "127.0.0.1:8888", "server listening address")
+	flag.BoolVar(&flags.captcha, "captcha", false, "enable captcha (default false)")
```

Comment:
```suggestion
	flag.StringVar(&flags.captchaSecret, "captcha-secret", "", "recaptcha secret (if empty, captcha are disabled)")
	flag.StringVar(&flags.faucetURL, "faucet-url", "http://localhost:5000" , "faucet server URL")
```

* As for the faucet, replace the captcha toggle by a captcha secret
* Add a faucet URL variable so that we can remove the hardcoded localhost/gno.land from the generated HTML

Reply:
Hello @moul, I loooove your suggestion approach.
However captcha secret key doesn't need here but site key does.

So I suggest
  1) main.go > replace from `BoolVar captcha` to `StringVar captchaSite`
  2) faucet.html > replace `data-sitekey="{YOUR_SITE_KEY}"` to `{{ if .Data.captchaSite }} data-sitekey={{ .Data.captchaSite }} {{ end }}`
  
How about that?